packages/create-gatsby/src/index.ts

```diff
@@ -119,7 +120,15 @@ export type PluginConfigMap = Record<string, Record<string, unknown>>
 
 const removeKey = (plugin: string): string => plugin.split(`:`)[0]
 
-export async function run(): Promise<void> {
+export const createCli = (): Enquirer<IAnswers> => {
+  const cli = new Enquirer<IAnswers>()
+
+  cli.use(plugin)
+
+  return cli
+}
+
+export async function run(cli: Enquirer<IAnswers>): Promise<void> {
```

Comment:
This is a breaking change. @gillkyle is using the run() export in gatsby new. Is there any way we can default to create it if it's empty? 

e.g. 
```typescript
export async function run(cli: Enquirer<IAnswers> = createCli()): Promise<void> 
```

Reply:
For sure we can 😊